### PR TITLE
Fixes #27401 - katello_repositories debug sql query fixed

### PR DESCRIPTION
**Pull request by lzap**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly

---

### CODE REVIEW

**Comment by jturel** [comment]
Looks good @lzap I think we need to bump the release in the spec file as well

**Comment by lzap** [comment]
Done!

File: packages/katello/katello/katello.spec [hunk 1]
```diff
@@ -194,6 +194,9 @@ Useful utilities for managing Katello services
 %{_sysconfdir}/bash_completion.d/katello-service
 
 %changelog
+* Tue Aug 13 2019 Lukas Zapletal <lzap+rpm@redhat.com> - 3.14.0-1.1.master
```
Comment (jturel) [inline packages/katello/katello/katello.spec]: It seems the release went out of sync with the version indicated in the changelog, possibly when 3.14 was bumped. @evgeni should this be 3.14.0-2.1.master now?

Comment (evgeni) [reply packages/katello/katello/katello.spec]: -0.2.master

Comment (lzap) [reply packages/katello/katello/katello.spec]: Updated the changelog

**Comment by jturel** [comment]
@evgeni pls2merge? :)

**Pull request merged**